hw_3/script.js

```diff
@@ -0,0 +1,111 @@
+function checkArgLength(args) {
+    if (args.length !== 2) {
+        throw new Error("Not 2 arguments");
```

Comment:
It would be better and clearer to expand the error description a little bit. As example:
```suggestion
        throw new Error("Function has a different number of arguments than two");
```

Reply:
Changed the error message. 
https://github.com/BlackTruth/javascript/blob/813db56a38899f41dcfdc4b332943bde5c271592/hw_3/script.js#L11